src/main/resources/localstack/s3/sites/sites.json

```diff
@@ -1,57 +0,0 @@
-[
-  {
-    "id": 999,
-    "name": "Special",
-    "enabled": true,
-    "clientTypes": ["DSP", "PUBLISHER", "DATA_PROVIDER", "ADVERTISER"]
-  },
-  {
-    "id": 123,
-    "name": "DSP",
-    "enabled": true,
-    "clientTypes": ["DSP"]
-  },
-  {
-    "id": 124,
-    "name": "Publisher",
-    "enabled": true,
-    "clientTypes": ["PUBLISHER"]
-  },
-  {
-    "id": 125,
-    "name": "Data Provider",
-    "enabled": true,
-    "clientTypes": ["DATA_PROVIDER"]
-  },
-  {
-    "id": 126,
-    "name": "Advertiser",
-    "enabled": true,
-    "clientTypes": ["ADVERTISER"]
-  },
-  {
-    "id": 127,
-    "name": "OptOut",
-    "enabled": true
-  },
-  {
-    "id": 128,
```

Comment:
Some of these are missing from the new mock data repo (e.g. 128-130) - I'm not sure how much other devs are relying on the test data that has been added here, but I know we've done some work to make sure the test data here matches other systems.

I haven't checked other JSON files, but we probably need to make sure the new test data is a superset of the various copies we're combining, otherwise it'll cause people a bunch of work to debug why their test data suddenly stopped working.